docs/api/v1.md

```diff
@@ -0,0 +1,117 @@
+# Spice API
+## Version `v1`
+
+All routes are prefixed with `/api/v1/`
```

Comment:
Do we need to version the routes if the frontend is always served by the backend? Anybody writing an alternative frontend should probably just fork the whole thing.